src/commands/synthetics/README.md

```diff
@@ -54,9 +54,9 @@ To setup the client, your Datadog API and application keys need to be configured
 
 ### Global configuration file options
 
-When you run your tests, use the `--config` flag on the command line to specify the path to the global configuration file.
+Create a JSON configuration file on your system. Specify the path to the file using the `--config` flag [when launching your tests](#run-tests). If you don't specify a file path, Datadog looks for a file with the default filename of `datadog-ci.json`.
```

Comment:
I see this text is exactly the same as line 53. Could you change that bullet point `3.`  to only redirect to the `#global-configuration-file-options` section?

Also, I think the paragraph could use a little introduction, instead of starting with "Create a JSON configuration file" out of nowhere

Reply:
Ah yeah this is a side effect of splitting the changes into multiple PRs 😅 on my other branch i've reordered the points and slightly changed them, removing this part from point `3.` 

https://github.com/DataDog/datadog-ci/pull/1372/files#diff-5a0fda3a46d1c105f78a54a8d87902bdb9886b1739928f72abc7cefc23a2f86bR38-R62